Projects/Team3/Andrii Nazaruk/NUnitTestLogin/HomePage.cs

```diff
@@ -0,0 +1,51 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using SeleniumExtras.PageObjects;
+using System.Threading;
+
+namespace NUnitTestLogin
+{
+    class HomePage
+    {
+        private IWebDriver crome_driver;
+        private IWebDriver firefox_driver;
+        private string url = "https://darknets.ru/";
+
+        public HomePage(IWebDriver _chrome_driver, IWebDriver _firefox_driver)
+        {
+            this.crome_driver = _chrome_driver;
+            this.firefox_driver = _firefox_driver;
+        }
+        public void goToPage()
+        {
+            crome_driver.Navigate().GoToUrl(url);
+            firefox_driver.Navigate().GoToUrl(url);
+            Thread.Sleep(500);
+        }
+        public void login()
+        {
+            firefox_driver.FindElement(By.XPath("/html/body/table[1]/tbody/tr/td[2]/form/table/tbody/tr[1]/td[2]/input")).SendKeys("Chadoboy");
+            firefox_driver.FindElement(By.CssSelector("#navbar_password")).SendKeys("g5FVajAy");
+            firefox_driver.FindElement(By.XPath("/html/body/table[1]/tbody/tr/td[2]/form/table/tbody/tr[2]/td[3]/input")).Click();
+
+            crome_driver.FindElement(By.XPath("/html/body/table[1]/tbody/tr/td[2]/form/table/tbody/tr[1]/td[2]/input")).SendKeys("Chadoboy");
+            crome_driver.FindElement(By.CssSelector("#navbar_password")).SendKeys("g5FVajAy");
+            crome_driver.FindElement(By.XPath("/html/body/table[1]/tbody/tr/td[2]/form/table/tbody/tr[2]/td[3]/input")).Click();
+            Thread.Sleep(3000);
```

Comment:
you no need to use thread.sleep
try to use 
driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(1000);